<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RpgPostWidget</class>
 <widget class="QWidget" name="RpgPostWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>415</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="posterLbl">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Poster&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="posterEdit">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="dateLbl">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Date&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="dateEdit">
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>30</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>30</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="contentLbl">
     <property name="minimumSize">
      <size>
       <width>70</width>
       <height>25</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>70</width>
       <height>25</height>
      </size>
     </property>
     <property name="text">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Content&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QTextEdit" name="contentEdit">
     <property name="minimumSize">
      <size>
       <width>320</width>
       <height>240</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
